djautotask/sync.py

```diff
@@ -52,8 +52,10 @@ def wrapper(*args, **kwargs):
                 f(*args, **kwargs)
             sync_job.success = True
         except Exception as e:
-            sync_job.message = str(e.args[0])
+            error_msg = f'Failed to create object. The error was: {e}'
```

Comment:
As an alternative - maybe we could create a `SyncJob` in the `_get_connection_url` method but **only** when there is an `AutotaskAPIError` exception. Does that make sense?

We wouldn't want to log when get_zone_info is successful because in those cases the `sync` method would run and that would create it's own sync logs.

Reply:
I realized `_get_connection_url` wasn't the best place for the code to log if we could complete the get_zone_info call. Mainly because `_get_connection_url` gets called for other things like the web URL which has nothing to do with syncing. I moved it to `get_api_connection_url` to avoid that. `get_api_connection_url` is only called when running the syncrhonizers.